### PR TITLE
49 db hosting

### PR DESCRIPTION
**Pull request by GD-Dheer**
This PR contains:

- Dockerized mysql service
- Modification of backend configuration to support mysql & typeORM
- Seeds the database
- Tests the connection between the backend and mysql via ORM
- Contains an example of table creation with TypeORM and verifies the entry in DB
- Contains an example of inserting elements in a table & finding an element in DB

No explicit testing & no strict typescript as this does not contain dev/transferrable code

### CODE REVIEW

File: Server/Dockerfile [hunk 1]
```diff
@@ -11,13 +11,16 @@ ADD package*.json ./
 RUN npm install
```
Comment (ds161833) [inline Server/Dockerfile:12]: comment about:
# pull official base image
FROM node:13.12.0-alpine	FROM node:13.12.0-alpine
why not use node 14

File: Server/wait-for-it.sh [hunk 2]
```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```
Comment (ds161833) [inline Server/wait-for-it.sh:1]: what is this script for?

File: Server/src/database.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import {createConnection} from 'typeorm';
+import {AnimeRank} from './models/entities/AnimeRank';
+
+export const connectDB = async () => {
+  await createConnection().then( async connection => {
```
Comment (ds161833) [inline Server/src/database.ts:5]: should be indented

File: Server/wait-for-it.sh (hunk 2 above)
Comment (GD-Dheer) [reply Server/wait-for-it.sh:1]: waits for db service to start before starting backend

File: Server/src/database.ts (hunk 3 above)
Comment (GD-Dheer) [reply Server/src/database.ts:5]: Which part?

Comment (ds161833) [reply Server/src/database.ts:5]:
```
await createConnection().then( async connection => {
  console.log("Connection was made.")
  //Methods for Entity Manager: https://github.com/typeorm/typeorm/blob/master/docs/entity-manager-api.md
  // This is using connection.manager.{methodName}``
```

File: Server/Dockerfile (hunk 1 above)
Comment (GD-Dheer) [reply Server/Dockerfile:12]: Updated

**Review by silent-po2 — APPROVED** [review]
Having resolved the issue regarding refreshing the database and the indentation fix, this appears good good to go.

**Review by Mehrdadmaskull — APPROVED** [review]

**Pull request merged**